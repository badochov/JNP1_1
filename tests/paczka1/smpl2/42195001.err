Error in line 1: A374 A374 39,0
Error in line 4: A374 S567 52,4
Error in line 8: J902G A805 14,4
Error in line 19: 1WUEkDISCn A374 80,5
Error in line 21: A374 A805 57,7
Error in line 27: J902G S567 12,2
Error in line 28: vdeu3oBi A805 308,8
