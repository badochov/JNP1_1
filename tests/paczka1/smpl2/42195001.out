S567 9,7
A792 36,9
A792 36,9
vdeu3oBi A 1,2 S 9,7
1WUEkDISCn A 36,9
